src/Core/Exception/InvalidException.php

```diff
@@ -0,0 +1,31 @@
+<?php
+/**
+ * 2007-2018 PrestaShop.
```

Comment:
Please update your licence headers. They now read "PrestaShop and Contributors" (see https://devdocs.prestashop.com/1.7/development/coding-standards/#core-files)

Also, the year is outdated 😛